### PR TITLE
Docker config

### PR DESCRIPTION
**Pull request by perploug**
As per #2  - includes needed files to launch probot in a docker container - still missing a proper delivery file

### CODE REVIEW

**Comment by perploug** [comment]
@fokusferit for your review

**Comment by perploug** [comment]
👍

File: delivery.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  - desc: Prepare
+    cmd: |
+      curl -o /tmp/ensure-docker -f https://delivery.cloud.zalando.com/utils/ensure-docker
+      chmod +x /tmp/ensure-docker
+      /tmp/ensure-docker
+
+  - desc: Build
+    cmd: |
+      function npm {
+        docker run -i --rm -v "$(pwd):/workspace" -w /workspace registry.opensource.zalan.do/stups/node:latest npm "$@"
```
Comment (fokusferit) [inline delivery.yaml]: Could you just explain me what this line is doing? some docker magic for me :D

Comment (fokusferit) [reply delivery.yaml]: Ok I see that this is used from the zappr part which as far as I understand to get the latest node + npm version. 
I should ask Rasha and others if we couldn't add this registry directly to docker and get rid of this line.

**Comment by fokusferit** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Comment by perploug** [comment]
👍

**Pull request merged**